packages/model/src/calendar.ts

```diff
@@ -72,6 +76,8 @@ export interface CalendarResponse {
   id: string;
   name: string;
   color: GoogleLegacyCalendarColor;
+  groups?: Pick<GroupDataObject, 'id' | 'active'>[];
```

Comment:
I don't think this makes much sesnse, we don't need groups or working-groups on the create method, especially not with both ID and Active properties. 

There is currently no requirement for creating calendars which associations and if you had looked at where this type is actually being used you would have found that the create method is not actually used anywhere while the update method of the data provider is exposed in a controller but only uses this input https://github.com/yldio/asap-hub/blob/14e72948d47f9b70ef1c1342cd9b732a232ea991/packages/model/src/calendar.ts#L85-L88